client/src/components/PaymentOptions.jsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState, useEffect } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Container from "@material-ui/core/Container";
+import Grid from "@material-ui/core/Grid";
+import Card from "@material-ui/core/Card";
+import CardContent from "@material-ui/core/CardContent";
+import Divider from "@material-ui/core/Divider";
+import SubscribeBtn from "../components/subscriptionBtn";
+import SubscribeDeleteBtn from "../components/subscriptionDeleteBtn";
+import axios from "axios";
+
+const PaymentOptions = () => {
+  const classes = useStyles();
+  const [url, setUrl] = useState(
+    window.location.pathname.replace("/upgrade", "").replace("/", "")
+  );
+  const [currentPlan, setCurrentPlan] = useState("free basic plan");
+  useEffect(() => {
+    axios
+      .get("http://localhost:3001/upgrade/checkSubscription", {
+        params: {
+          calendarUrl: url,
+        },
+      })
+      .then((response) => {
+        console.log("Response", response.data.plan.amount);
+        if (response.data.plan.amount == 1999) {
+          setCurrentPlan("professional plan");
+        }
+        if (response.data.plan.amount == 999) {
```

Comment:
```suggestion
        else if (response.data.plan.amount == 999) {
```

this will save on a extra logic check :)